app/controllers/scientist_experiments_controller.rb

```diff
@@ -0,0 +1,12 @@
+class ScientistExperimentsController < ApplicationController
+  def destroy
+    scientist = Scientist.find(params[:id])
+    experiments = scientist.experiments
+    experiment = experiments.find(params[:id])
+    scientist_experiment = scientist.scientist_experiments.find_by(scientist_id: scientist.id, experiment_id: experiment.id)
+
+    scientist_experiment.destroy
+
+    redirect_to scientist_path(scientist)
+  end
```

Comment:
You're using `params[:id]` to find both the Scientist and Experiment. Do they share the same ID?